server/src/services/util.es6

```diff
@@ -263,23 +263,21 @@ util.isProduction = () => {
 };
 
 /**
- * @function isStaging - is staging flag
- * @return {boolean} - NODE_ENV is staging
+ * @function isTestProductionAuthenticationEnabled - should production
+ * authentication be used in tests (used in CI for more realistic test
+ * conditions)
+ * @return {boolean} - true when test production authentication is enabled
  */
-util.isStaging = () => {
-  return process.env.NODE_ENV === 'staging';
+util.isTestProductionAuthenticationEnabled = () => {
+  return !!process.env.TEST_PRODUCTION_AUTH;
```

Comment:
Yeah, `!!` because it's just looking for a truthy value and it should convert it to a boolean. Note that if you set `TEST_PRODUCTION_AUTH=0` it still evaluates to true but I think that's okay and keeps the logic simple.

No matter the environment, it's just based on the environment variable. So in production if you set this variable, it will apply. Hmm, we could add an additional check `NODE_ENV === 'test'` so it can never be enabled when its not supposed to.

Reply:
Oh, i guess it doesn't really matter because it's only used in the localAuth middleware which is not enabled in production. Also test production auth really means production auth, so it's not like enabling it is bad. I'll add the check though, can't hurt and it's semantically correct since it is a test configuration setting.